File: AgroGuard-Frontend/src/api/dashboard-api/agricultureData.ts
import service from 'request'

// @ts-ignore
export function apiGetAgricultureData({region, commodity, year, item, period} = {}) {
  return service({
    url: `/dashboard/supply-demand/agriculture-data`,
    method: 'get',
    params: { region, commodity, year, item, period }
  })
}

File: AgroGuard_Backend/src/main/java/com/api/agroguard/entity/AgricultureData.java
package com.api.agroguard.entity;

import lombok.Data;

@Data
public class AgricultureData {
    private String code;
    private String report_month;
    private String region;
    private String commodity;
    private String item;
    private String year;
    private String period;
    private Double value;
    private Double min_value;
    private Double max_value;
}
